pages/advanced-search.html

```diff
@@ -363,17 +363,62 @@ <h1>Builder</h1>
       <section class="search-details">
         <h1>Details</h1>
         <p>For more information about a search, enter a query above and click &ldquo;show details.&rdquo;</p>
+        <span class="notice">
+          <span class="label-text">Loading&hellip;</span>
+          <span class="icon icon-loading"></span>
+        </span>
         <dl>
           <dt class="search-details--results">Results</dt>
           <dd class="search-details--results">1,946</dd>
           <dt class="search-details--search-expansion">Search expansion</dt>
-          <dd class="search-details--search-expansion">"myocardial infarction"[MeSH Terms] OR ("myocardial"[All Fields] AND "infarction"[All Fields]) OR "myocardial infarction"[All Fields] OR ("heart"[All Fields] AND "attack"[All Fields]) OR "heart attack"[All Fields]</dd>          
+          <dd class="search-details--search-expansion">"myocardial infarction"[MeSH Terms] OR ("myocardial"[All Fields] AND "infarction"[All Fields]) OR "myocardial infarction"[All Fields] OR ("heart"[All Fields] AND "attack"[All Fields]) OR "heart attack"[All Fields]</dd>
         </dl>
       </section>
     </div>
     <section class="search-history">
       <h1>History</h1>
       <p>Your history is currently empty! As you use PubMed, your recent queries will show up here to help you build more focused searches.</p>
+      <h1>History</h1>
+      <div class="search-history--actions actions">
```

Comment:
You have disassociated both the table label and actions from the table.

Reply:
This isn't synced with master is it? Because this table isn't in advanced search (and shouldn't be)